src/organisms/sensors.py

```diff
@@ -4,15 +4,15 @@ def __init__(self, size):
         self.translation = {"danger" : 0, "empty" : 0.4, "food" : 1}
         self.org_translation = {"danger" : 0, "empty" : 0.6, "food" : 0.8}
```

Comment:
What are we doing about that now? I don't think we really came to a conclusion on this.

Reply:
I feel like it worked out, the biggest issue i feel like is them not comprehending the edge of the world